src/toil/jobStores/fileJobStore.py

```diff
@@ -162,24 +164,29 @@ def jobs(self):
     # Functions that deal with temporary files associated with jobs
     ##########################################
 
+    def _copyOrLink(self, url, path):
+        # linking is not done be default because of issue #1755
+        if self.linkImports:
+            try:
+                os.link(os.path.realpath(self._extractPathFromUrl(url)), path)
```

Comment:
@arkal actually the error only occured when trying to overwrite the original file with the new output by using exportFile. Changing the permissions here won't help with reason 2. (Reason 1 is enough for me though).